wetterdienst/ui/core.py

```diff
@@ -56,25 +56,25 @@ def _get_stations_request(
     date: Optional[str],
     issue: str,
     si_units: bool,
-    tidy: bool,
+    shape: Literal["long", "wide"],
     humanize: bool,
     skip_empty: bool,
     skip_threshold: float,
     skip_criteria: Literal["min", "mean", "max"],
     dropna: bool,
-    use_nearby_station_until_km: float,
+    use_nearby_station_distance: float,
 ):
     from wetterdienst.provider.dwd.mosmix import DwdMosmixRequest, DwdMosmixType
 
     settings = Settings(
-        si_units=si_units,
-        tidy=tidy,
-        humanize=humanize,
-        skip_empty=skip_empty,
-        skip_criteria=skip_criteria,
-        skip_threshold=skip_threshold,
-        dropna=dropna,
-        interp_use_nearby_station_until_km=use_nearby_station_until_km,
+        scalar_si_units=si_units,
+        scalar_shape=shape,
+        scalar_humanize=humanize,
+        scalar_skip_empty=skip_empty,
+        scalar_skip_criteria=skip_criteria,
+        scalar_skip_threshold=skip_threshold,
+        scalar_dropna=dropna,
+        scalar_interpolation_use_nearby_station_distance=use_nearby_station_distance,
```

Comment:
Maybe we can do something like this, in order to bundle the `scalar` settings into another settings object, and compose them?

On the "naming things" side, would it make sense to use the notion of `DataSettings`, or even `TransformationSettings`?

```suggestion
        scalar=ScalarSettings(
            si_units=si_units,
            shape=shape,
            humanize=humanize,
            skip_empty=skip_empty,
            skip_criteria=skip_criteria,
            skip_threshold=skip_threshold,
            dropna=dropna,
            interpolation_use_nearby_station_distance=use_nearby_station_distance,
        )
```

Reply:
Scalar would refer to the ScalarRequest calss but maybe PointRequestCore and PointSettings would make even more sense.

Reply:
In my opinion this would add more complexity then just prefixing... Does it hurt anyway to add a small prefix to the option?

Reply:
Like see if I only want the parameters to be not humanized:

```Python
Settings(point_humanize=False)
```

but then I'd have

```Python
Settings(
    point=PointSettings(
        humanize=False
    )
)
```

If it was more settings to be defined then I'd agree but at the current level I think it doesnt make sense.

Reply:
> I like the idea that we copy basically the names from the env variables, otherwise it would add more complexity.

I agree with you. Having a flat list of settings, and having them represented by unique labels which can be discovered throughout the code base is very fortunate.